js/components/ConsortReport.vue

```diff
@@ -27,8 +27,8 @@
       </div>
 
       <div v-if="hasReportSummaries">
-        <ReportSummary v-for="summary in reportSummaries"
-                       :key="summary.reportId"
+        <ReportSummary v-for="(summary, i) in reportSummaries"
+                       :key="summary.reportId + i"
```

Comment:
It's currently possible to add a summary for the same report more than once. This prevents a duplicate key warning, although we might want to prevent duplicate summaries.